packages/task/src/browser/task-service.ts

```diff
@@ -203,13 +202,13 @@ export class TaskService implements TaskConfigurationClient {
 
     /** Returns an array of the task configurations configured in tasks.json and provided by the extensions. */
     async getTasks(): Promise<TaskConfiguration[]> {
-        const configuredTasks = this.getConfiguredTasks();
+        const configuredTasks = await this.getConfiguredTasks();
         const providedTasks = await this.getProvidedTasks();
         return [...configuredTasks, ...providedTasks];
     }
 
     /** Returns an array of the task configurations which are configured in tasks.json files */
-    getConfiguredTasks(): TaskConfiguration[] {
+    getConfiguredTasks(): Promise<TaskConfiguration[]> {
```

Comment:
Does this require a breaking change note?
Clients who call the following method will at least need to be `async` now.